src/components/blocks/eventsBlock/EventsBlock.tsx

```diff
@@ -26,7 +26,7 @@ export const EventsBlock = ({ families }: IProps) => {
   useEffect(() => {
     const language = navigator?.language;
 
-    setUpdatedRowsWithLocalisedDates(getEventTableRows({ families, language }));
+    setUpdatedRowsWithLocalisedDates(getEventTableRows({ families, language, isLitigation: true }));
```

Comment:
Gripe: I hate this pattern but it's well beyond the scope of this PR, so one for me to noodle over later.